target/linux/ramips/dts/mt7621_humax_e10.dts

```diff
@@ -0,0 +1,153 @@
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/mtd/partitions/uimage.h>
+
+/ {
+	compatible = "humax,e10", "mediatek,mt7621-soc";
+	model = "HUMAX QUANTUM E10";
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+		label-mac-device = &wan;
+	};
+	
+	gpio-export {
+		compatible = "gpio-export";
+
+		gpio_usb_power {
+                        gpio-export,name = "humax:power:usb";
+                        gpio-export,output = <1>;
+                        gpios = <&gpio 8 GPIO_ACTIVE_HIGH>;
+                };
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power: power {
+			label = "humax:green:power";
```

Comment:
We don't use "humax:" vendor prefix anymore. It should be "green:power".

Reply:
Removed!